README.md

```diff
@@ -118,13 +128,15 @@ In this example we just pass the jdbc connection url and set ssl to false. The f
              },
 ```
 In the final section of the request body is where we set the final configuration properties.
-`metadataSourceQualifiedName` specifies the Egeria qualified name of the entity that represents the Software Server entity represents the postgres server.
-If you have already created and entity to represent the postgres server simply enter the qualified name here. If you haven't already created the server
+`metadataSourceQualifiedName` specifies the Egeria qualified name of the entity that represents the Software Server entity represents the Postgres server.
```

Comment:
'that represents the Software Server entity represents the ' 	doesn't make sense? Is it the Software Server entity?
Maybe just say
'the Egeria qualified name of the Software Server entity that represents the Postgres server'

Reply:
reworded this in new PR